upb/decode.c

```diff
@@ -608,6 +609,25 @@ static bool decode_tryfastdispatch(upb_decstate *d, const char **ptr,
   return false;
 }
 
+static const char *decode_mset_item(upb_decstate *d, const char *ptr,
```

Comment:
mset is another name we should improve, especially given the existence of memset

Reply:
Done.